graylog2-web-interface/src/components/permissions/GranteesSelector.tsx

```diff
@@ -129,6 +128,7 @@ const GranteesSelector = ({ availableGrantees, availableCapabilities, className,
                                     onChange={(granteeId) => onChange({ target: { value: granteeId, name } })}
                                     optionRenderer={_renderGranteesSelectOption}
                                     options={granteesOptions}
+                                    // @ts-ignore
```

Comment:
Is there a different way to solve this?

Reply:
With https://github.com/Graylog2/graylog2-server/pull/9713 I changed this part and this `@ts-ignore` should no longer be necessary. I will remove it one of these pull requests.